src/transformers/models/bert/tokenization_bert.py

```diff
@@ -171,7 +171,7 @@ class BertTokenizer(PreTrainedTokenizer):
 
             This should likely be deactivated for Japanese (see this
             [issue](https://github.com/huggingface/transformers/issues/328)).
-        strip_accents (`bool`, *optional*):
+        strip_accents (`str`, *optional*):
```

Comment:
Are you sure this is a `str`? The relevant code is in `super()` but it feels like a `bool` to me.